instrumentation/reactor-netty/reactor-netty-1.0/javaagent/src/test/groovy/AbstractReactorNettyHttpClientTest.groovy

```diff
@@ -179,4 +188,42 @@ abstract class AbstractReactorNettyHttpClientTest extends HttpClientTest<HttpCli
     assert expectedSpanContext.traceId == actualSpanContext.traceId
     assert expectedSpanContext.spanId == actualSpanContext.spanId
   }
+
+  // custom address resolver that returns at most one address for each host
+  // (because reactor-netty will retry on connection failure across multiple resolved addresses
+  // which creates spans that are not expected by the http client tests)
+  // adapted from io.netty.resolver.DefaultAddressResolverGroup
+  static class CustomNameResolverGroup extends AddressResolverGroup<InetSocketAddress> {
```

Comment:
I'm assuming this is because of our test that hits `google.com`. I got burned by this a little while ago for not being able to pass tests in offline mode when developing in a waiting room so am planning on rewriting this test relatively soon. No big deal just FYI

Reply:
Actually for this test it was localhost that resolved both to ipv4 and ipv6 address.